include/dpp/guild.h

```diff
@@ -492,17 +516,17 @@ class DPP_EXPORT guild : public managed, public json_interface<guild>  {
 	/** Number of boosters */
 	uint16_t premium_subscription_count;
 
-	/** Maximum users in a video channel, or 0 */
-	uint16_t max_video_channel_users;
+	/** Voice AFK timeout in seconds before moving users to AFK channel. Can be set to: 60, 300, 900, 1800, 3600 */
+	uint16_t afk_timeout;
 
-	/** Voice AFK timeout before moving users to AFK channel */
-	uint8_t afk_timeout;
+	/** Maximum users in a video channel, or 0 */
+	uint8_t max_video_channel_users;
```

Comment:
the afk_timeout field is now uint16_t & max_video_channel_users is now uint8_t, just because it's enough